guide/contribute/review.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: guide
+title: Review
+description: Contribute to the guide by reviewing a proposed change
+nav_order: 2
+parent: Contribute to guide
+permalink: /guide/contribute/review
+image: /assets/images/guide/contribute/contribute-review-preview.jpg
```

Comment:
Image does not exist.

Reply:
Correct, I've done that on purpose, Jose told me he'll work on those and I'll add them as soon as they're ready.